config/checkstyle_checks.xml

```diff
@@ -114,6 +114,13 @@
     <property name="fileExtensions" value="xml, vm"/>
     <property name="message" value="Line should not be longer than 100 symbols"/>
   </module>
+  <!-- links to .dtd files should start with "/", "http://" or "https://" -->
+  <module name="RegexpSingleline">
+    <property name="format" value="href=&quot;(?!\/|https?:\/\/).*?\.dtd&quot;"/>
+    <property name="fileExtensions" value="xml, vm"/>
+    <property name="message"
+              value="Relative links to DTD files are prohibited. Please prepend it with a slash."/>
```

Comment:
`Please prepend it with a slash`
Does `Please use absolute path or uri instead.`  sounds better?

Reply:
done